Source/WebCore/svg/SVGLength.cpp

```diff
@@ -32,7 +32,31 @@ namespace WebCore {
 
 ExceptionOr<float> SVGLength::valueForBindings()
 {
-    return m_value.valueForBindings(SVGLengthContext { RefPtr { contextElement() }.get() });
+    RefPtr element = contextElement();
+    // Ensure style is up-to-date for font-relative units (e.g., ch).
+    // https://www.w3.org/TR/css-values-3/#absolute-lengths
+    auto lengthType = m_value.lengthType();
```

Comment:
nit: we just use this variable once and the name doesn't document anything. I guess we don't need it.

Reply:
I think that would be better but this is very not important